tools/backup/create-azure-secret.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+NAMESPACE=<YOUR_NEO4J_NAMESPACE>
+CREDENTIALS_SECRET=<YOUR_NEO4J_NAMESPACE>
+
+CREDENTIALS=$(cat <<-END
+export ACCOUNT_NAME=<NAME_STORAGE_ACCOUNT>
+export SUBSCRIPTION=<SUBSCRIPTION_NAME_OR_ID>
+export SP_ID=<SERVICE_PRINCIPAL_NAME_OR_ID>
+export SP_PASSWORD=<GENERATED_SERVICE_PRINCIPAL_SECRET>
+export TENANT_ID=<SERVICE_PRINCIPAL_NAME_OR_ID>
+END
+)
+CREDENTIALS=$(echo $CREDENTIALS | base64)
+
+cat >neo4j-backup-secret.yaml <<EOL
```

Comment:
one of your questions dealt with where to put documentation:   the docs for backups are here:

https://github.com/neo4j-contrib/neo4j-helm/blob/master/doc/docs/modules/ROOT/pages/backup.adoc

In this particular bit of code, you're doing something differently than what the docs already recommend, which is kubectl to create a secret from a file.   I'd recommend having a look there, and following the same pattern already recommended for creating "credentials" secrets, so the pattern is always the same no matter the cloud.

Reply:
Removed this file and moved to docs folder.